templates/zerver/integrations/jitsi.md

```diff
@@ -1,17 +1,34 @@
-[Jitsi Meet](https://jitsi.org/jitsi-meet/) is Zulip's default video
-call provider. Jitsi Meet is a fully-encrypted, 100% open source video
-conferencing solution.
+By default, Zulip integrates with [Jitsi Meet](https://jitsi.org/jitsi-meet/),
+a fully-encrypted, 100% open source video conferencing solution.
+
+Zulip uses the [cloud version of Jitsi Meet](https://meet.jit.si/)
+as its default video call provider. You can also use a self-hosted
+instance of Jitsi Meet.
 
 ### Using Jitsi Meet
 
-1. Set Jitsi as the organization's [video call
-   provider](/help/start-a-call#changing-your-organizations-video-call-provider),
-   if it isn't already.
+{settings_tab|organization-settings}
+
+1. Under **Other settings**, select **Jitsi Meet** from the **Call provider**
+   dropdown.
+
+1. _(optional)_ Select **Custom URL** from the **Jitsi server URL** dropdown,
+   and enter the URL of your self-hosted Jitsi Meet server. You can also set
+   `JITSI_SERVER_URL` in `/etc/zulip/settings.py` to specify the server's URL.
+
+1. Click **Save changes**, and use <kbd>Esc</kbd> to exit the settings.
+
+1. [Open the compose box](/help/open-the-compose-box), and click the
+   **video camera** (<i class="fa fa-video-camera"></i>) icon at the
+   bottom of app. This will insert a **Join video call.** link into
```

Comment:
How about noting the voice call option here, at least, so that we don't have to think about it later? Something like this should do:

1. [Open the compose box](/help/open-the-compose-box), and click the
   **video camera** (<i class="fa fa-video-camera"></i>) icon to insert a video call link, or the **phone** (icon) to insert an audio call link into your message.